### PR TITLE
Add workshop powerplatform

### PR DESCRIPTION
**Pull request by olivMertens**
For reference 30Days platform workshop

### CODE REVIEW

File: database/external.yml [hunk 1]
```diff
@@ -58,4 +58,20 @@
...
\ No newline at end of file
+  tags: api, apim, azure, devops, policy, gateway
+
+- title: Workshop Powerplatform
+  url: https://microsoft.github.io/Low-Code/docs/intro/
+  github_url: https://github.com/microsoft/Low-Code
+  language: en
+  last_updated: 2023-02-15
+  type: workshop
+  description: This hands-on Workshop  low code application development workshops that can help you skill up with self-guided tutorials and real-world applications. Be sure to check back regularly for updates!.
```
Comment (sinedied) [inline database/external.yml:69]: Phrase is weird :D

**Review by sinedied — CHANGES_REQUESTED** [review]

File: database/external.yml (hunk 1 above)
Comment (olivMertens) [reply database/external.yml:69]: yeah , yoghurt copy/paste 👎🏻

Comment (olivMertens) [reply database/external.yml]: Must be good now

**Review by jonathanmf — APPROVED** [review]
Adding the workshop

**Review by sinedied — APPROVED** [review]

**Pull request merged**